### PR TITLE
Add packaged zip checksum files to existing ant task

### PR DESCRIPTION
**Pull request by cbridgha**

### CODE REVIEW

**Comment by cbridgha** [comment]
#build

File: dev/wlp-generateChecksums/src/com/ibm/ws/wlp/cs/GenerateZipChecksums.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+    private void createFile(File file, Properties props) throws IOException {
+        OutputStream out = null;
+        try {
+            out = new FileOutputStream(file, false);
+            props.store(out, null);
+        } finally {
+            FileUtils.tryToClose(out);
+        }
+    }
```
Comment (mbroz2) [inline dev/wlp-generateChecksums/src/com/ibm/ws/wlp/cs/GenerateZipChecksums.java:87]: Dead code?

**Review by mbroz2 — APPROVED** [review]

**Comment by cbridgha** [comment]
#build

**Comment by cbridgha** [comment]
#libby

**Pull request merged**